packages/admin/cms-admin/src/common/SearchInput.tsx

```diff
@@ -93,16 +93,16 @@ export const SearchInput = ({
             endAdornment={
                 internalQuery ? (
                     <InputAdornment position="end">
-                        <Typography>
```

Comment:
The spacing between the number and the arrow-icon is missing

| Design | Implementation |
| --- | --- |
| <img width="545" alt="Design" src="https://github.com/user-attachments/assets/581ba7e7-a157-4607-aaf7-1c83a0c08c23" /> | <img width="545" alt="Implementation" src="https://github.com/user-attachments/assets/5c12ec19-db11-45bb-8cd8-b03ed1536c07" /> |